README.md

```diff
@@ -9,3 +9,16 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+A bijection is established when each node in $G_{1}$ uniquely maps to a comparable node in $G_{2}$. Given that $G_{1}$ and $G_{2}$ contain the same number of nodes, we can define a mapping function _f_ that creates this one-to-one correspondence. For the mapping function to exist we have to prove that there is a one-to-one correspondence and onto. For the one-to-one, we know that no unique node from $G_{1}$ is connected to two $G_{2}$ nodes. There is one unique node from $G_{1}$ that is connected to a node from $G_{2}$. Which has to pass the one-to-one. Then onto is that every node from $G_{1}$ and $G_{2}$ are paired to leave no pairs out making it onto. 
```

Comment:
You're still not proving that this mapping exists -- you're saying about what properties this mapping would have if it were to exist. How do I know that I can construct such a mapping?